### DISCUSSION TITLE
QCefView能否实现在CEF中的打印预览功能？

### DISCUSSION THREAD

**Discussion by yufewng** [CefView/QCefView#140] [Q&A] [answered]
如题，在CEF能实现html的打印预览功能，在QCefView中是否也能实现此功能？

**Comment by yufewng** [comment, marked as answer]
问题解决了
在CMakeLists.txt文件中将
option(USE_OSR          "Enable CEF Offscreen rendering"      ON)
改为
option(USE_OSR          "Enable CEF Offscreen rendering"      OFF)
就可以使用打印预览功能，我猜是CEF在离线渲染模式下无法开启打印预览的原因。

**Reply by 1139870644** [reply]
可是也想在OSR下开启打印预览，而且调用浏览器内核的打印预览和直接调用window的打印设置，两个保存下来的pdf有些是不一样的
